src/helics/core/CoreBroker.hpp

```diff
@@ -330,11 +331,11 @@ class CoreBroker: public Broker, public BrokerBase {
     /** check if we can remove some dependencies*/
     void checkDependencies();
     /** find any existing publishers for a subscription*/
-    void FindandNotifyInputTargets(BasicHandleInfo& handleInfo);
-    void FindandNotifyPublicationTargets(BasicHandleInfo& handleInfo);
+    void FindAndNotifyInputTargets(BasicHandleInfo& handleInfo, const std::string& key);
+    void FindAndNotifyPublicationTargets(BasicHandleInfo& handleInfo, const std::string& key);
 
-    void FindandNotifyFilterTargets(BasicHandleInfo& handleInfo);
-    void FindandNotifyEndpointTargets(BasicHandleInfo& handleInfo);
+    void FindAndNotifyFilterTargets(BasicHandleInfo& handleInfo, const std::string& key);
+    void FindAndNotifyEndpointTargets(BasicHandleInfo& handleInfo, const std::string& key);
```

Comment:
If we want to match the naming convention used by other functions, the first letter of these function names should be lowercase.